src/rust/engine/process_execution/src/remote_cache.rs

```diff
@@ -512,23 +512,27 @@ impl crate::CommandRunner for CommandRunner {
     };
 
     if result.exit_code == 0 && self.cache_write {
-      let write_result = self
-        .update_action_cache(
-          &context,
-          &request,
-          &result,
-          &self.metadata,
-          &command,
-          action_digest,
-          command_digest,
-        )
-        .await;
-      if let Err(err) = write_result {
-        log::warn!("Failed to write to remote cache: {}", err);
-        context
-          .workunit_store
-          .increment_counter(Metric::RemoteCacheWriteErrors, 1);
-      };
+      let command_runner = self.clone();
+      let result = result.clone();
```

Comment:
Thanks @stuhood for helping me to figure out why we needed to do this to avoid an issue with non-static lifetimes in the future!

Lmk if I should comment these two lines.